Findovsrpro.cmake

```diff
@@ -4,6 +4,8 @@
 # should match xpGetCompilerPrefix in externpro's xpfunmac.cmake
 # NOTE: wanted to use externpro version, but chicken-egg problem
 function(getCompilerPrefix _ret)
+  set(options GCC_TWO_VER)
+  cmake_parse_arguments(x "${options}" "" "" ${ARGN})
```

Comment:
this file shouldn't even exist in this repo -- it should be generated at build-time (or actually cmake-time) -- there is a template in externpro https://github.com/smanders/externpro/blob/18.01.1/modules/Findscript.cmake.in that creates this file and I can't think of any good reason to commit it to the repo